rts/Lua/LuaVAOImpl.cpp

```diff
@@ -185,6 +194,116 @@ int LuaVAOImpl::AddObjectsToSubmissionImpl(const sol::stack_table& ids)
 	return submitCmds.size() - idsSize;
 }
 
+template<typename TObj>
+void LuaVAOImpl::Bins::ModifyImpl(const sol::stack_table& removedObjects, const sol::stack_table& addedObjects,
+	sol::optional<size_t> removedCount, sol::optional<size_t> addedCount,
+	std::vector<SDrawElementsIndirectCommand>& submitCmds)
+{
+	const size_t removedObjectCount = (removedCount.has_value()? removedCount.value() : removedObjects.size());
+	const size_t addedObjectCount = (addedCount.has_value()? addedCount.value() : addedObjects.size());
+	if (removedObjectCount == 0 && addedObjectCount == 0) return;
+
+	size_t firstChangedBinIndex = -1;
```

Comment:
This tripped me up a bit until I saw it was size_t, better go for `std::numeric_limits<size_t>::max()`

Reply:
Ok